src/OrchardCore.Modules/OrchardCore.Forms/Views/ToolFormWorkflow.cshtml

```diff
@@ -0,0 +1,41 @@
+@using OrchardCore.DisplayManagement.Zones
+
+@{
+    var tabs = (IEnumerable<string>)ZoneShapes.HarvestAndSortTabs(Model.Content);
```

Comment:
What is `Model.Content` here? How do you target the `Tabs` ?

Reply:
It's your magic! I just defined it as `CotentZone` and I could use placement to tabify

Reply:
Haha, what I meant was what is placed in this Model.Content zone, is it a wrapper for the form content widgets, or ...

Reply:
I created `ToolFormWorkflow` shape as placeholder shape for workflow selection tool button - applicable to only `Form` content type - injected in toolbar by Form module when enabled. 

Idea is that clicking on this  button will open a side bar which would allow user to select workflow for the form (not included in PR). Content is placed via `OnDisplay`  below from zone `WorkflowSettings`

https://github.com/OrchardCMS/OrchardCore/blob/5c4a638c687ce84560f44eff8eb97a7f5b30c15f/src/OrchardCore.Modules/OrchardCore.Forms/Shapes.cs#L41-L56

`WorkflowSettings` Zone and it's content is defined in `placement.json` 

https://github.com/OrchardCMS/OrchardCore/blob/5c4a638c687ce84560f44eff8eb97a7f5b30c15f/src/OrchardCore.Modules/OrchardCore.Forms/placement.json#L50-L57

Such a way that  if Tabs are defined in placement then it will show tabs.